### PR TITLE
Added createRedirectUrl to client and fixed timestamp issue on JWT's

### PR DESCRIPTION
**Pull request by matthisk**
#61 and #58 both fixed. Used 'qs' library to create querystring. createRedirectUrl fails on incorrect targetUrl (similar to python implementation. Created two tests to test createRedirectUrl one to check whether wrong target urls result in an error and one to test the result. Notice that I decode the resulting JWT and check that result, instead of checking against an encoded JWT (like the python lib does). That is a bad idea because the order of the payload json affects the encoded JWT making the test really prone to failures (that do not origin from anything actually breaking) especially since dicts do not have a true ordering. Maybe a good idea to change this test in the py lib as well.

### CODE REVIEW

**Comment by tschellenbach** [comment]
That's a good point, very slick implementation.

File: src/lib/client.js [hunk 1]
```diff
@@ -50,6 +53,7 @@ StreamClient.prototype = {
     this.group = this.options.group || 'unspecified';
     // track subscriptions made on feeds created by this client
     this.subscriptions = {};
+    this.noJWTTimestamp = this.options.noJWTTimestamp || true;
```
Comment (tbarbugli) [inline src/lib/client.js]: hard to set noJWTTimestamp to `false` here ;)

Comment (tbarbugli) [inline src/lib/client.js]: I would rename noJWTTimestamp (keep JWT out of it) to something like expireTokens or something similar.

**Comment by matthisk** [comment]
Good catch, fixed the issue and renamed to expireTokens.

**Pull request merged**